docs/serving/app-security/security-guard-quickstart.md

```diff
@@ -0,0 +1,174 @@
+
+        An easy way to do that is using:
+
+        ```
+        kubectl apply -f https://github.com/knative-sandbox/security-guard/releases/download/v0.1.0/queue-proxy.yaml
+        ```
+
+    1. Add the necessary Guard resources to your cluster using:
+
+        ```
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/release-0.1/config/resources/gateAccount.yaml
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/release-0.1/config/resources/serviceAccount.yaml
+        kubectl apply -f https://raw.githubusercontent.com/knative-sandbox/security-guard/release-0.1/config/resources/guardiansCrd.yaml
+        ```
+
+    1. Deploy `guard-service` on your system to enable automated learning of micro-rules. In the current version, it is recommended to deploy `guard-service` in any namespace where knative services are deployed.
+
+        An easy way to do that is using:
+
+        ```
+        kubectl apply -f https://github.com/knative-sandbox/security-guard/releases/download/v0.1.0/guard-service.yaml
+        ```
+
+## Creating and deploying a service
+
+!!! tip
+
+    The following commands create a `helloworld-go` sample Service while activating and configuring the `security-guard` extension for this Service. You can modify these commands, including changing the `security-guard` configuration for your service using either the `kn` CLI or changing the service yaml based on this example.
+
+Create a sample securedService:
+
+=== "Apply YAML"
+
+    1. Create a YAML file using the following example:
+
+        ```yaml
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        metadata:
+          name: helloworld-go
+          namespace: default
+        spec:
+          template:
+            metadata:
+                annotations:
+                  features.knative.dev/queueproxy-podinfo: enabled
+                  qpoption.knative.dev/guard-activate: enable
+            spec:
+              containers:
+                - image: gcr.io/knative-samples/helloworld-go
+                  env:
+                    - name: TARGET
+                      value: "Secured World"
+        ```
+
+    1. Apply the YAML file by running the command:
+
+        ```
+        kubectl apply -f <filename>.yaml
+        ```
+
+        Where `<filename>` is the name of the file you created in the previous step.
+
+=== "kn CLI"
+
+    ```
+    kn service create helloworld-go \
+        --image gcr.io/knative-samples/helloworld-go \
+        --env "TARGET=Secured World" \
+        --annotation features.knative.dev/queueproxy-podinfo=enabled \
+        --annotation qpoption.knative.dev/guard-activate=enable
+    ```
+
+After the Service has been created, Guard starts monitoring the Service Pods and all Events sent to the Service.
+
+## Example alerts
```

Comment:
I would make this its own reference page in the docs in the left nav, it doesn't seem like it's really part of the quickstart because it's not clear what a user is achieving with each step. This should be clarified or this section should be moved to a reference doc page.